test/ese/src/blue/src/esetest/esetest/sources.no.autogenerate

```diff
@@ -0,0 +1 @@
+a
```

Comment:
just an "a" ?

Reply:
Was this the file you thought got accidentally included and you wanted to take out?

Reply:
Yeah, I'm deleting it. And yes, it just contains the letter "a". I think it's just a file that, if it exists, some autogeneration doesn't occur, judging by the name.